config/env/production.js

```diff
@@ -44,6 +44,12 @@ module.exports = {
 		clientSecret: process.env.GITHUB_SECRET || 'APP_SECRET',
 		callbackURL: '/api/auth/github/callback'
 	},
+  paypal: {
+      clientID: process.env.PAYPAL_ID || 'CLIENT_ID',
+      clientSecret: process.env.PAYPAL_SECRET || 'CLIENT_SECRET',
+      callbackURL: '/api/auth/paypal/callback',
+      sandbox: false
```

Comment:
@ilanbiala I see that in the strategy, it has an option to pass sandbox option for testing with the sandbox environment instead of production so you don't actually charge the paypal account.
https://github.com/shekhei/passport-paypal-openidconnect/blob/master/lib/passport-paypal-openidconnect/strategy.js#L54

if true, it will use the sandbox urls:
https://github.com/shekhei/passport-paypal-openidconnect/blob/master/lib/passport-paypal-openidconnect/strategy.js#L12